apps/vr-tests/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "6.0.0",
   "private": true,
   "scripts": {
-    "build": "",
+    "build": "node ../../scripts/build.js ts --commonJsOnly",
```

Comment:
@kenotron might need to be updated when we switch to `just`

Reply:
acknowledged - I'll wait until this PR is in before I merge mine #7415